clusters/veritable-prod/alice/ui/values.yaml

```diff
@@ -25,7 +23,14 @@ postgresql:
 ingress:
   hostname: alice.vrtbl.com
   ingressClassName: nginx-alice
+emailTransport: SMTP_EMAIL
+smtpHost: smtp.azurecomm.net
+smtpPort: 587
+emailFromAddress: "alice@vrtbl.com"
```

Comment:
I assume this doesn't exist? We probably want an admin email address at the minimum that we can get to. Can you check with Esther what she wants?

Reply:
I do not understand what you mean.

Reply:
How will we receive the admin email

Reply:
We don't, this is a send-from address as far as I know.

Reply:
The `emailFromAddress` is a send-from address. `emailAdminAddress` is actually sent email and we need to be able to receive this. It should probably be a secret in this repo and go to a mailing list we control